services/web/server/src/simcore_service_webserver/director_v2_handlers.py

```diff
@@ -136,3 +136,33 @@ async def stop_pipeline(request: web.Request) -> web.Response:
             reason=exc.reason,
             http_error_cls=get_http_error(exc.status) or web.HTTPServiceUnavailable,
         )
+
+
+@routes.get(f"/{VTAG}/computations/{{project_id}}")
+@login_required
+@permission_required("services.pipeline.*")
+@permission_required("project.read")
+async def get_pipeline(request: web.Request) -> web.Response:
+    # @PC: I'll need your help here
+    client = DirectorV2ApiClient(request.app)
+    run_policy = get_project_run_policy(request.app)
```

Comment:
Yep! Here is idfficult to say since the model we return here (that should represent a Computation resource) is not the same as the one we return in the start ... 
To answer this we need to analyze more carefully definitions here and  how the meta-modeling policy adapts

Reply:
actually they are the same.